app/src/main/java/uk/ac/ebi/atlas/search/JsonGeneSearchController.java

```diff
@@ -179,6 +137,68 @@ public String search(@RequestParam MultiValueMap<String, String> requestParams)
                         "checkboxFacetGroups", ImmutableList.of(MARKER_GENE.getTitle(), ORGANISM.getTitle())));
     }
 
+    @RequestMapping(value = "/json/search/marker-genes",
```

Comment:
Rather than “overloading” the `/json/seach` let’s add a new `/json/gene-search` path. Initially we only searched genes, so there was no ambiguity bout the endpoint, but now, since we can also search metadata terms, I think it’s better if we split it.

Reply:
OK, I changed it, but I am still not convinced that we need the `/json` prefix in the path.
I left it there for now.